api/tests/integration/infrastructure/repositories/knowledge-element-repository_test.js

```diff
@@ -198,14 +198,13 @@ describe('Integration | Repository | KnowledgeElementRepository', () => {
       const userId_tmp = databaseBuilder.factory.buildUser().id;
 
       _.each([
-        { skillId: 'rec1', userId, earnedPix: 5, competenceId: 1 },
-        { skillId: 'rec3', userId, earnedPix: 40, status: 'validated', competenceId: 1 },
-        { skillId: 'rec2', userId, earnedPix: 10, status: 'validated', createdAt: today, competenceId: 1 },
-        { skillId: 'rec4', userId, earnedPix: 10, status: 'validated', competenceId: 2 },
+        { skillId: 'rec1', userId, earnedPix: 5.0, competenceId: 1 },
+        { skillId: 'rec3', userId, earnedPix: 20.1, status: 'validated', competenceId: 1 },
```

Comment:
```suggestion
        { skillId: 'rec3', userId, earnedPix: 20.4, status: 'validated', competenceId: 1 },
```
Pour vérifier le cas de ne pas prendre en compte les virgules j'augmenterais ce KE.